src/perform_step/rosenbrock_perform_step.jl

```diff
@@ -736,16 +736,17 @@ end
     integrator.k[1] =  h21*k1 + h22*k2 + h23*k3 + h24*k4 + h25*k5
     integrator.k[2] =  h31*k1 + h32*k2 + h33*k3 + h34*k4 + h35*k5
   end
+  integrator.fsallast = du
   integrator.u = u
 end
 
 
-function initialize!(integrator, cache::Rodas4Cache)
-  integrator.kshortsize = 2
-  @unpack dense1,dense2 = cache
-  resize!(integrator.k, integrator.kshortsize)
-  integrator.k .= [dense1,dense2]
-end
+# function initialize!(integrator, cache::Rodas4Cache)
```

Comment:
?

Reply:
this function is equivalent to the already defined `
function initialize!(integrator, cache::Union{...})` n L276, except that `integrator.fsallast` and `integrator.fsalfirst` are not defined here and remain `undef`. So firstly it is redundant and secondly it does not initialize completely

Reply:
these algorithms don't want/have/need FSAL?

Reply:
This is how it looks to me. Anyway it does not hurt much to keep them.

Reply:
Yeah I think they are needed since these algorithms aren't FSAL so fsalfirst and fsallast are unused on purpose.

Reply:
I have to admit that I'am not much in to all the details. I just looked up how it was done in the working `Rodas5()` and did some copy-paste instead of waiting for an low priority issue to get a fix.